README.md

```diff
@@ -91,6 +91,29 @@ Further information about the configuration of the sampler is provided in the `R
 
 **Note:** In all configuration files, the paths to input files from the previous stages are adjusted automatically. Also cross-parameters that require consistency between the hydrodynamics evolution and the sampler, e.g. viscosities and critical energy density, are taken care of automatically.
 
+
+## Using a custom SMASH configuration file for the initial conditions
+
+In addition to the above described predefined Au+Au and Pb+Pb collisions, it is possible to employ a custom `SMASH` configuration file in the initial stage. This file is expected to be located in the  `[...]/configs/` directory and named `smash_initial_conditions_custom.yaml`. An example configuration is shipped with the `SMASH-vHLLE-hybrid`, it can be modified as desired. To run the hybrid with this custom configuration file, execute the following commands:
+
+    make custom_IC
+    make custom_hydro
+    make custom_sampler
+    make custom_afterburner
+
```

Comment:
I'm afraid this would imply an entire restructuring of the hybrid itself and is not easily doable... 
The implementation of the Zr and Ru collisions is however anyways planned differently, such that they can be run simultaneously, no worries to this end ;)